test/routes.post.test.js

```diff
@@ -0,0 +1,100 @@
+const request = require('supertest');
+const app = require('../src/backend/web/app');
+const Post = require('../src/backend/post');
+
+describe('test /post responses', () => {
+  // an array of keys.
+  const existingGuid = 'existing-guid';
+  const missingGuid = 'missing-guid';
+  const existingNeedingEncoding = 'guid needing/encoding';
+  const missingNeedingEncoding = 'missing needing/encoding';
+
+  // an object to be added for testing purposes
+  const addedPost1 = new Post(
+    'foo',
+    'foo',
+    '',
+    'foo',
+    new Date('2009-09-07T22:20:00.000Z'),
+    new Date('2009-09-07T22:23:00.000Z'),
+    'foo',
+    'foo',
+    existingGuid
+  );
+
+  // an object, expected to be returned by a correct querry
```

Comment:
`query` (sp).

Reply:
Thank you very much @humphd! My bad with the spelling 😅 
Should I fix this?

Reply:
Don't feel badly, I can't spell either.  I just use tools to hide it https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker :)

Yes, please fix this and I'll merge.